### PR TITLE
Set GC rule for requestKey

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- Set GC rule for requestKey (max: 20, deleted: 10)
- Fix typo in write rule

Related issues:
- https://github.com/ainize-team/ainize-js/issues/117
- https://github.com/ainize-team/ainize-js/issues/118

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for handling it!

File: src/ainize.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default class Ainize {
    */
   async getService(serviceName: string): Promise<Service> {
     const servicePath = Path.app(serviceName).root();
-    const serviceData = await this.ain.getValue(servicePath);
+    const serviceData = await this.ain.getValue(servicePath, { is_shallow: true });
```
Comment (jiyoung-an) [inline src/ainize.ts:110]: I add `is_shallow` option, please check if other functions work well.

**Comment by platfowner** [comment]
@yoojinko @akaster99 could you review & test this pr?

**Comment by yoojinko** [comment]
LGTM, Thanks. It works well. Sorry for my late review.

**Review by yoojinko — APPROVED** [review]

**Comment by platfowner** [comment]
@yoojinko Thanks for testing!

**Comment by jiyoung-an** [comment]
LGTM, Thank you for review! now merging...

**Pull request merged**